crates/frontend/src/pages/default_config/default_config.rs

```diff
@@ -5,7 +5,9 @@ use crate::components::skeleton::Skeleton;
 use crate::components::stat::stat::Stat;
 use crate::components::table::{table::Table, types::Column};
 use crate::types::BreadCrums;
-use crate::utils::unwrap_option_or_default_with_error;
+use crate::utils::{
+    get_local_storage, set_local_storage, unwrap_option_or_default_with_error,
```

Comment:
`unwrap_option_or_default_with_error` is unused ?

Reply:
No it's being used.